activesupport/test/logger_test.rb

```diff
@@ -16,6 +16,21 @@ def setup
     @logger  = Logger.new(@output)
   end
 
+  def with_level(level = Logger::INFO)
+    begin
```

Comment:
Methods are enclosed in their own begin scope, so you don't need to call begin and ensure can be at the same indentation as def.